AdobeStockAsset/etc/db_schema.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="adobe_stock_request" resource="default" engine="innodb" comment="Adobe Stock License Request">
+        <column xsi:type="int" name="id" padding="10" unsigned="true" nullable="false" identity="true" comment="Entity ID"/>
+        <column xsi:type="int" name="admin_user_id" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Admin User Id"/>
+        <column xsi:type="varchar" length="255" name="request_id" nullable="false" comment="Request ID"/>
+        <constraint xsi:type="primary" referenceId="PRIMARY">
+            <column name="id"/>
+        </constraint>
+        <constraint xsi:type="foreign" referenceId="ADO_STO_LIC_REQ_ADM_USER_ID" table="adobe_stock_request" column="admin_user_id" referenceTable="admin_user" referenceColumn="user_id" onDelete="CASCADE"/>
+    </table>
+    <table name="adobe_stock_language" resource="default" engine="innodb" comment="Adobe Stock Languages">
+        <column xsi:type="int" name="language_id" padding="10" unsigned="true" nullable="false" identity="true" comment="Entity ID"/>
+        <column xsi:type="varchar" length="255" name="language" nullable="false" comment="Languages"/>
+        <constraint xsi:type="primary" referenceId="PRIMARY">
+            <column name="language_id"/>
+        </constraint>
+    </table>
+    <table name="adobe_stock_user_language" resource="default" engine="innodb" comment="Adobe Stock User Language">
+        <column xsi:type="int" name="language_id" padding="10" unsigned="true" nullable="false" identity="true" comment="Language ID"/>
```

Comment:
There should be no `identity="true"` fields in this table as each column can have multiple entries with the same value.
```suggestion
        <column xsi:type="int" name="language_id" padding="10" unsigned="true" nullable="false" identity="false" comment="Language ID"/>
```